test/acceptance/behave/README.md

```diff
@@ -21,15 +29,11 @@ Note: We recommend the use of virtualenv, because is an isolated working copy of
 - Both if you are using a virtual environment or not:
```

Comment:
The `virtualenv ~/venv` and `source ~/venv/bin/activate` should be shown in code font type (as in this comment).